grpc/service_config/service_config.proto

```diff
@@ -191,7 +191,11 @@ message MethodConfig {
 }
 
 // Configuration for pick_first LB policy.
-message PickFirstConfig {}
+message PickFirstConfig {
+  // If set to true, instructs the LB policy to shuffle the list of addresses
```

Comment:
Maybe change this to say "randomly shuffle"?  I want to be clear that the shuffling is not deterministic.

Reply:
Done. Thanks.